resources/geocoding/en/54.txt

```diff
@@ -13,224 +13,332 @@
 # limitations under the License.
 
 # Translated from es/54.txt using Freebase and Wikipedia.
+# https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina
 # Some manual edits were made after the file was generated, according to the
 # ITU file published 2011-12-16 here: http://www.itu.int/oth/T0202000009/en
 
 5411|Buenos Aires
-54220|Merlo
-54221|La Plata
-542221|Verónica, Buenos Aires
-542223|Coronel Brandsen
-542224|Buenos Aires Province
+54114200|Quilmes, Buenos Aires
+54114224|Quilmes, Buenos Aires
+54114250|Quilmes, Buenos Aires
+54114251|Quilmes, Buenos Aires
+54114252|Quilmes, Buenos Aires
+54114253|Quilmes, Buenos Aires
+54114254|Quilmes, Buenos Aires
+54114255|Florencio Varela, Buenos Aires
+54114256|Berazategui, Buenos Aires
+54114257|Quilmes, Buenos Aires
+54114258|Berazategui, Buenos Aires
+54114259|Quilmes, Buenos Aires
+54114278|Quilmes, Buenos Aires
+54114350|Quilmes, Buenos Aires
+54114546|Autonomous City of Buenos Aires
+54114858|Autonomous City of Buenos Aires
+54114919|Autonomous City of Buenos Aires
+54220|Merlo, Buenos Aires
+542202|La Matanza, Buenos Aires
```

Comment:
2202 is given as González Catán , Viceroy of the Pine in wikipedia. La Matanza is not mentioned anywhere in the whole doc.